docker/docker-compose.yml

```diff
@@ -1,12 +1,30 @@
 version: "3.9"
 services:
+  endorser-caddy:
+    image: caddy
+    hostname: endorser-caddy
+    environment:
+      - CADDY_PORT=${CADDY_PORT}
+    ports: # Uncomment to access caddy outside of containers
+      - ${CADDY_PORT}:${CADDY_PORT}
+      # - 2019:2019
+    volumes:
+      - ./caddy/Caddyfile:/etc/caddy/Caddyfile:z
+    healthcheck:
+      # Port 2019 is an internal Caddy admin port.
+      test: nc -zv localhost:2019 || exit -1
```

Comment:
You should be able to use an environment variable for the port rather than hard coding the number.

Reply:
Fixed